### PR TITLE
Feature review

### PR DESCRIPTION
**Pull request by Makolin0**
Changed usage of react router, renamed pages files, added dev navbar with useful tools, added css variables for font size, spacing and colors.

### CODE REVIEW

File: src/components/Review.jsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+                                                src={
+                                                    reviewData.rating >= index
+                                                        ? "../../public/star.png"
+                                                        : "../../public/star-empty.png"
+                                                }
+                                            />
+                                        </button>
+                                    </li>
+                                );
+                            })}
```
Comment (kacper-dabrowski) [inline src/components/Review.jsx]:
```suggestion
                            {[...Array(5)].map((a, index) => {
                            const rating = index + 1
                                return (
                                    <li key={rating}>
                                        <button
                                            type="button"
                                            onClick={() =>
                                                ratingButtonHandler(rating)
                                            }
                                        >
                                            <img
                                                src={
                                                    reviewData.rating >= rating
                                                        ? "../../public/star.png"
                                                        : "../../public/star-empty.png"
                                                }
                                            />
                                        </button>
                                    </li>
                                );
                            })}
```

File: src/components/Review.jsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                            <input
+                                id="tip"
+                                type="number"
+                                name="tip"
+                                onChange={(event) =>
+                                    inputHandler("tipAmount", event)
+                                }
+                                value={reviewData.tipAmount}
+                            />
+                        </div>
```
Comment (kacper-dabrowski) [inline src/components/Review.jsx]: 1. Looks like a separate component
2. Could be done with map (like stars above)
example:
const tips = [0,5,10,20]

tips.map(tip=>...)

Comment (Makolin0) [reply src/components/Review.jsx]: fixed

File: src/components/Review.jsx (hunk 1 above)
Comment (Makolin0) [reply src/components/Review.jsx]: applied adjustment

**Review by ThePapiok — APPROVED** [review]

**Review by DarthRevan77797 — APPROVED** [review]

**Pull request merged**